src/frontend/api/fagsak.ts

```diff
@@ -60,6 +61,13 @@ export const hentFagsaker = (
     );
 };
 
+export const søkFagsaker = (personIdent: string): Promise<Ressurs<IFagsakDeltager[]>> => {
+    return axiosRequest({
+        method: 'GET',
+        url: 'familie-ba-sak/api/fagsaker/sok?personIdent=' + personIdent,
```

Comment:
We should never have pin in URL. They will be logged to kibana etc. Put it in a header :)

Reply:
ok